lib/widgets/user_profile.dart

```diff
@@ -120,12 +107,30 @@ class UserProfile extends HookWidget {
                           ?.jwt
                           .raw,
                     ))
-                    .then((val) => val.comments),
+                    .then((val) => val.posts),
```

Comment:
Is this an intentional change? Or another bad merge?

Reply:
I think this might be what has broken the user profile view if you try to go between comments and posts.

Reply:
If you try scrolling down the list of posts or comments, it does this weird snap back to the top thing.

Reply:
I believe the above described issue is actually caused by this: https://github.com/liftoff-app/liftoff/pull/287